docker-compose.yml

```
@@ -0,0 +1,54 @@
+version: "3.3"
+services:
+  db:
+    image: postgres:14
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=decidim_bulletin_board
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+  bulletin:
```

I had to double check that this was actually another service.

Also, I prefer to use the `app` service naming as its more standard (it could be `web`, whatever you prefer, as long is not `bulletin` 😄)  
```suggestion
      - pg-data:/var/lib/postgresql/data

  app:
```

---

Probably `web`  is better as that's the one used in the heroku.yml